lib/resourcemerge/rbacv1beta1.go

```diff
@@ -1,6 +1,7 @@
 package resourcemerge
```

Comment:
We might actually be able to drop this entirely, with:

```console
$ oc adm release extract --to manifests quay.io/openshift-release-dev/ocp-release:4.8.0-fc.3-x86_64
$ grep -r10 v1beta1 manifests/ | grep ClusterRole
manifests/0000_80_machine-config-operator_03_rbac.yaml-kind: ClusterRoleBinding
manifests/0000_30_baremetal-operator_01_baremetalhost.crd.yaml-kind: ClusterRole
```

The MCO has since [bumped their version][1] and the baremetal manifest [doesn't have annotations to be included in any cluster profiles][2].  So I expect we can drop this entirely, instead of fixing v1beta1 hotlooping.

[1]: https://github.com/openshift/machine-config-operator/pull/2572
[2]: https://bugzilla.redhat.com/show_bug.cgi?id=1956607

Reply:
Part of dropping v1beta1 RBAC will be:

```console
$ git --no-pager diff -U0 hack
diff --git a/hack/generate-lib-resources.py b/hack/generate-lib-resources.py
index 441e253d..294e9885 100755
--- a/hack/generate-lib-resources.py
+++ b/hack/generate-lib-resources.py
@@ -271 +270,0 @@ if __name__ == '__main__':
-    types['k8s.io/api/rbac/v1beta1'] = types['k8s.io/api/rbac/v1']
@@ -282 +280,0 @@ if __name__ == '__main__':
-        'k8s.io/api/rbac/v1beta1': {'package': 'k8s.io/client-go/kubernetes/typed/rbac/v1beta1', 'type': 'RbacV1beta1Client'},
```

although you'll also want #552, and you'll need to drop some remaining v1beta1 RBAC code like this resourcemerge stuff by hand.

Reply:
Good idea, dropped v1beta1